foo=>_await(_catch(foo,()=>"fallback"))